docs/virtual_machines/istio_service_mesh.md

```diff
@@ -0,0 +1,147 @@
+kind: NetworkAttachmentDefinition
+metadata:
+  name: istio-cni
+```
+
+## Create a VirtualMachineInstance with enabled Istio proxy injecton
+
+The example below specifies a VMI with masquerade network interface and `sidecar.istio.io/inject` annotation to register the VM to the service mesh. 
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  annotations:
+    sidecar.istio.io/inject: "true"
+  labels:
+    app: vmi-istio
+  name: vmi-istio
+spec:
+  domain:
+    devices:
+      interfaces:
+        - name: default
+          masquerade: {}
+      disks:
+        - disk:
+            bus: virtio
+          name: containerdisk
+    resources:
+      requests:
+        memory: 1024M
+  networks:
+    - name: default
+      pod: {}
+  terminationGracePeriodSeconds: 0
+  volumes:
+    - name: containerdisk
+      containerDisk:
+        image: registry:5000/kubevirt/fedora-cloud-container-disk-demo:devel
+```
+
+Istio expects each application to be associated with at least one Kubernetes service. Create the following Service exposing port 8080:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: vmi-istio
+spec:
+  selector:
+    app: vmi-istio
+  ports:
+    - port: 8080
+      name: http
+      protocol: TCP
+```
+
+**Note:** Each Istio enabled VMI must feature the `sidecar.istio.io/inject` annotation instructing KubeVirt to perform necessary network configuration.
+
+## Verification
+
+Verify istio-proxy sidecar is deployed and able to synchronize with Istio control plane using `istioctl proxy-status` command. See Istio [Debbuging Envoy and Istiod](https://istio.io/latest/docs/ops/diagnostic-tools/proxy-cmd/) documentation section for more information about `proxy-status` subcommand.
+
+```shell
+$ kubectl get pods
+NAME                             READY   STATUS    RESTARTS   AGE
+virt-launcher-vmi-istio-ncx7r    3/3     Running   0          7s
+
+$ kubectl get pods virt-launcher-vmi-istio-ncx7r -o jsonpath='{.spec.containers[*].name}'
+compute volumecontainerdisk istio-proxy
+
+$ istioctl proxy-status
+NAME                                    CDS        LDS        EDS        RDS          ISTIOD                      VERSION
+...
+virt-launcher-vmi-istio-ncx7r.default   SYNCED     SYNCED     SYNCED     SYNCED       istiod-7c4d8c7757-hshj5     1.10.0
+```
+
+## Troubleshooting
+
+1. `istio-proxy` sidecar is not deployed in VMI virt-launcher pod:
+
+```shell
+$ kubectl get pods
+NAME                             READY   STATUS    RESTARTS   AGE
+virt-launcher-vmi-istio-jnw6p    2/2     Running   0          37s
+
+$ kubectl get pods virt-launcher-vmi-istio-jnw6p -o jsonpath='{.spec.containers[*].name}'
+compute volumecontainerdisk
+```
+
+**Resolution:** Make sure the `istio-injection=enabled` is added to the target namespace. If the issue persists, consult [relevant part of Istio documentation](https://istio.io/latest/docs/ops/configuration/mesh/injection-concepts/).
+
+2. `istio-proxy` sidecar is not Ready:
```

Comment:
This renders as 1. again due to it being parsed as a new list: https://deploy-preview-466--kubevirt-user-guide.netlify.app/virtual_machines/istio_service_mesh/#istio-service-mesh